freqtrade/rpc/telegram.py

```diff
@@ -862,12 +863,39 @@ def _send_msg(self, msg: str, parse_mode: str = ParseMode.MARKDOWN,
         :return: None
         """
 
+        # default / fallback shortcut buttons
         keyboard: List[List[Union[str, KeyboardButton]]] = [
             ['/daily', '/profit', '/balance'],
             ['/status', '/status table', '/performance'],
             ['/count', '/start', '/stop', '/help']
         ]
 
+        # do not allow commands with mandatory arguments and critical cmds
+        # like /forcesell and /forcebuy
+        # TODO: DRY! - its not good to list all valid cmds here. But this
+        #       needs refacoring of the whole telegram module (same problem
+        #       in _help()).
+        valid_btns: List[str] = ['/start', '/stop', '/status', '/status table',
+                                 '/trades', '/profit', '/performance', '/daily',
+                                 '/stats', '/count', '/locks', '/balance',
+                                 '/stopbuy', '/reload_config', '/show_config',
+                                 '/logs', '/whitelist', '/blacklist', '/edge',
+                                 '/help', '/version']
+        # custom shortcuts specified in config.json
+        cust_keyboard = self._config['telegram'].get('keyboard', [])
+        if cust_keyboard:
+            # check for valid shortcuts
+            invalid_keys = [b for b in chain.from_iterable(cust_keyboard)
+                            if b not in valid_btns]
+            if len(invalid_keys):
+                logger.warning('rpc.telegram: invalid commands for custom '
+                               f'keyboard: {invalid_keys}')
+                logger.info('rpc.telegram: using default keyboard.')
```

Comment:
I would think that we might want to do this check in `__init__()` and prevent startup if you configure something wrong...? 

otherwise you start the bot - but if you don't carefully read the logs, you won't notice you have a typo ...